src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php

```diff
@@ -266,6 +266,8 @@ public function viewAction($customerId, Request $request)
             'note' => $customerInformation->getGeneralInformation()->getPrivateNote(),
         ]);
 
+        $this->manageLegacyFlashes($request->query->get('conf'));
```

Comment:
```suggestion
        if ($request->query->has('conf')) {
            $this->manageLegacyFlashes($request->query->get('conf'));
        }
```

And you can remove the empty check in `manageLegacyFlashes`.
Do not trigger unwanted calls.
Plus, since there is a check for `$messages[$messageId]` the `empty($messageId)` is not needed again

Reply:
Fixed